src/condor_ce_view

```diff
@@ -92,7 +92,10 @@ def parse_opts():
     opts = parser.parse_args()[0]
 
     if hasattr(htcondor, 'set_subsystem'):
-        htcondor.set_subsystem("CEVIEW")
+        try:
+            htcondor.set_subsystem(kwargs['localname'])
+        except KeyError:
+            htcondor.set_subsystem("CEVIEW")
```

Comment:
It _seems_ like the intention here is that `localname` is an optional argument.  In that case, the above works, but (to me anyway) it's not clear whether the try/except for a `KeyError` is for `kwargs` or the `set_subsystem` call, or both.

If it's just for `kwargs`, you could test it with `if 'localname' in kwargs` instead of the try/except, or just do a `dict.get` with a default:
```htcondor.set_subsystem(kwargs.get('localname', 'CEVIEW'))```

But if `localname` is the only optional argument for `parse_opts`, you could also just define the function that way with `def parse_opts(localname=None)` and then later test if `localname is None` or be slightly lazy (but not dangerously) and do something like:
```htcondor.set_subsystem(localname or "CEVIEW")```